tests/test_writing_methods.py

```diff
@@ -44,9 +94,12 @@ def test_051_add_account_and_sync_wrong_and_correct_pin(figo_session):
         task_state = figo_session.add_account_and_sync("de", wrong_credentials, BANK_CODE)
     except FigoPinException as pin_exception:
         task_state = figo_session.add_account_and_sync_with_new_pin(pin_exception, CREDENTIALS[1])
-    time.sleep(5)
-    assert isinstance(task_state, TaskState)
-    assert len(figo_session.accounts) == 3
+        assert isinstance(task_state, TaskState)
+        assert len(figo_session.accounts) == 3
+    except FigoException as figo_exception:
```

Comment:
i'm more of a fan of `pytest.raises` but no idea if that works with multiple error classes ... that raises the question: why can this here raise `FigoPinException` or  `FigoException` ?